cashapp/stagehand
# Update CI for latest macOS and Xcode

Updates CI and the build script:

- Use `macos-latest` for latest macOS version and toolchain
- Replace package generate (which no longer exists) with a simple `swift build`
- Remove Bazel build support, bazel users should consume via https://registry.bazel.build/modules/rules_swift_package_manager.
- Remove Xcode job, the `pod lib lint` job still exists and continues to validate the podspec which is what we care about.
- Remove `build.swift` and instead update the github workflow to run the commands via Bash, this is a lot simpler and does the same in less code.
